recipe-server/client/control/components/MultiPicker.js

```diff
@@ -209,9 +208,9 @@ class PickerControl extends React.Component {
 export default class MultiPicker extends React.Component {
   static propTypes = {
     unit: pt.string.isRequired,
+    onChange: pt.func.isRequired,
```

Comment:
I was getting a warning in the console saying the given propType(s) were undefined, so I switched it out with a normal React type. I think the [redux-form propTypes](https://github.com/erikras/redux-form/blob/master/src/propTypes.js) are provided for the form component, not the fields.